R/render.R

```diff
@@ -4,7 +4,10 @@
 #' (knitted from R Markdown) and JavaScript code to render slides.
 #' \code{tsukuyomi()} is an alias of \code{moon_reader()}.
 #'
-#' Tsukuyomi is a genjutsu to trap the target in an illusion on eye contact.
+#' @details Tsukuyomi is a genjutsu to trap the target in an illusion on eye contact.
+#' @details If you are unfamiliar with CSS, please see the
```

Comment:
You don't really need the `@details` tags here. The convention of roxygen is: the first paragraph is the title, the second paragraph is the description, and the rest of paragraphs are details.